components/network/memcached/Makefile

```diff
@@ -81,8 +81,8 @@ COMPONENT_TEST_TRANSFORMS += \
 TEST_REQUIRED_PACKAGES += library/perl-5/io-socket-ssl-538
 
```

Comment:
Please replace that with
TEST_REQUIRED_PACKAGES.perl += library/perl-5/io-socket-ssl
This should create a dependency on the default perl version (5.40) of this package.

Reply:
done